templates/zerver/app/index.html

```diff
@@ -141,7 +141,7 @@
 </div>
 
 <div id="tooltip-templates-container"></div>
-<div id="streams_overlay_container"></div>
+<div id="channels_overlay_container"></div>
```

Comment:
Seemed like an easy refactor/clean-up for the stream/channel rename to do while I was focused on this. And made it easier to double-check that any `#streams` remaining in the codebase were not related to this overlay hash change.

Reply:
Yep, makes sense to me!